addons/block_code/simple_nodes/simple_scoring/simple_scoring.gd

```diff
@@ -19,6 +19,13 @@ const _POSITIONS_FOR_PLAYER = {
 }
 
 
+func _init():
+	var node = preload("res://addons/block_code/simple_nodes/simple_scoring/_simple_scoring.tscn").instantiate() as Node
+	node.replace_by(self, true)
```

Comment:
When I play the Pong game, it shows error:
```
E 0:00:00:0452   simple_scoring.gd:24 @ _init(): Condition "p_node->data.parent" is true.
  <C++ Source>   scene/main/node.cpp:2796 @ replace_by()
  <Stack Trace>  simple_scoring.gd:24 @ _init()
                 block_code.gd:56 @ _update_parent_script()
                 block_code.gd:14 @ _ready()
```

Reply:
Ooh, I see how that happened. That's fixed in the newest set of commits.